core/template.yaml

```
@@ -244,8 +261,27 @@ Resources:
               Bool:
                 aws:SecureTransport: false
 
-Outputs:
+  PublishKeyFunction:
+    Type: AWS::Serverless::Function
+    Metadata:
+      BuildMethod: esbuild
+      BuildProperties:
+        Sourcemap: true
+    Properties:
+      Handler: core/publish-key/src/publish-key-handler.lambdaHandler
+      FunctionName: !Sub "${AWS::StackName}-publishKey"
+      Environment:
+        Variables:
+          POWERTOOLS_SERVICE_NAME: !Sub "${CriIdentifier}-publishKey"
+      Timeout: 15
+
```

a manage tag is required to make changes to the bucket e.g. like [this](https://github.com/govuk-one-login/identity-key-rotation/blob/main/template.yaml#L584C1-L586C26)
````suggestion
Tags:
        - Key: "key_consumer_type"
          Value: "manage"
````